java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -987,6 +987,26 @@ public void addCoreML(EnumSet<CoreMLFlags> flags) throws OrtException {
       addCoreML(OnnxRuntime.ortApiHandle, nativeHandle, OrtFlags.aggregateToInt(flags));
     }
 
+    /**
+     * Adds Xnnpack as an execution backend. Needs to list all options here if a new option
+     * supported. current supported options: {}
+     *
+     * @param provider_options options pass to XNNAPCK EP for initialization.
+     * @throws OrtException If there was an error in native code.
+     */
+    public void addXnnpack(Map<String, String> provider_options) throws OrtException {
```

Comment:
Variable names & arguments in the Java code should be `camelCase` not `snake_case`.

Reply:
fixed